github_pr.py

```diff
@@ -59,7 +59,7 @@ def _print_pr(pr, **args):
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title.encode('ascii', errors='ignore'))
+        print "#%d [%s] %10s:%s <- %s:%-30s    %s" % (pr.number, pr.state, pr.base.repo.owner.login, pr.base.ref, pr.head.repo.owner.login, pr.head.ref, pr.title.encode('ascii', errors='ignore'))
```

Comment:
I think this is a needed change. I think incrementing the major version is excessive, let's make it 2.1. I think you want this as the string though, to keep the pretty spacing:
```
"#%d [%s] %s:%10s <- %-30s:%s    %s"
```

I think we should add the forks as extra columns in the tabular output